doc/material/index.rst

```diff
@@ -78,14 +88,22 @@ This command line only builds the scenario but does not solve it. To solve the s
     mix-models --url="ixmp://ixmp_dev/MESSAGEix-Materials/scenario_name" material-ix \
      SSP2 solve --add_calibration False --add_macro False
 
-The solve command has the `--add_calibration` option to add MACRO calibration to a baseline scenario. `--add_macro` option solves the scenario with MACRO.
-Both options are False by default.To first calibrate the scenario and then solve that scenario with MACRO both options should be set to True.
+The solve command has the ``--add_calibration`` option to add MACRO calibration to a baseline scenario with a valid calibration file specified with ``--macro-file``.
+The ``--add_macro`` option determines whether the scenario should be solved with MESSAGE or MESSAGE-MACRO.
+MESSAGEix-Materials provides one calibration file that is only compatible with scenarios with first model year 2025 and the common model structure of a MESSAGEix-GLOBIOM scenario.
+To first calibrate the scenario and then solve that scenario with MACRO both options should be set to True.
+
+It is also possible to shift the first model year and solve a clone with shifted years with ``--shift_model_year``.
+If ``--shift_model_year`` is set together with the macro options the model year will be shifted before the MACRO calibration.
+
+All three options are False by default.
+
 
 Reporting
 =========
 
 The reporting generates specific variables related to materials, mainly Production and Final Energy.
-The resulting reporting file is generated under message_ix_models\\data\\material\\reporting_output with the name “New_Reporting_MESSAGEix-Materials_scenario_name.xlsx”.
+The resulting reporting file is generated under :file:`message_ix_models\\data\\material\\reporting_output` with the name “New_Reporting_MESSAGEix-Materials_scenario_name.xlsx”.
```

Comment:
```suggestion
The resulting reporting file is generated under :file:`message_ix_models/data/material/reporting_output` with the name “New_Reporting_MESSAGEix-Materials_scenario_name.xlsx”.
```

Please use the same style for paths we use everywhere else (which is the Linux/macos/unix style, much more comfortable to users of QWERTY keyboards than the windows style).